pcdsdevices/fms.py

```diff
@@ -0,0 +1,259 @@
+        self.elevation = elevations
+        super().__init__(*args, **kwargs)
+
+#LCP FEE ALCOVE TEMP
+
+class LCP1(Device):
+   Temperature_F = Cpt(EpicsSignalRO, ':Temperature_CALC', kind='hinted')
+   Power_Vsupply_V = Cpt(EpicsSignalRO, ':24Vsupply_CALC', kind='hinted')
+   AirFan1Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan1Rpm_CALC', kind='hinted')
+   AirFan2Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan2Rpm_CALC', kind='hinted')
+   AirFan3Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan3Rpm_CALC', kind='hinted')
+   AirFan4Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan4Rpm_CALC', kind='hinted')
+   AirFan5Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan5Rpm_CALC', kind='hinted')
+   AirFan6Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan6Rpm_CALC', kind='hinted')
+   WaterTempIn_F = Cpt(EpicsSignalRO, ':WaterTempIn_CALC', kind='hinted')
+   WaterTempOut_F = Cpt(EpicsSignalRO, ':WaterTempOut_CALC', kind='hinted')
+   WaterFlowRate_gpm = Cpt(EpicsSignalRO, ':WaterFlowRate_CALC', kind='hinted')
+   AirTempInTop_F = Cpt(EpicsSignalRO, ':AirTempInTop_CALC', kind='hinted')
+   AirTempInMid_F = Cpt(EpicsSignalRO, ':AirTempInMid_CALC', kind='hinted')
+   AirTempInBot_F = Cpt(EpicsSignalRO, ':AirTempInBot_CALC', kind='hinted')
+   AirTempOutTop_F = Cpt(EpicsSignalRO, ':AirTempOutTop_CALC', kind='hinted')
+   AirTempOutMid_F = Cpt(EpicsSignalRO, ':AirTempOutMid_CALC', kind='hinted')
+   AirTempOutBot_F = Cpt(EpicsSignalRO, ':AirTempOutBot_CALC', kind='hinted')
+  
+class LCP2(Device):
+   Temperature_F = Cpt(EpicsSignalRO, ':Temperature_CALC', kind='hinted')
+   Power_Vsupply_V = Cpt(EpicsSignalRO, ':24Vsupply_CALC', kind='hinted')
+   AirFan1Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan1Rpm_CALC', kind='hinted')
+   AirFan2Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan2Rpm_CALC', kind='hinted')
+   AirFan3Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan3Rpm_CALC', kind='hinted')
+   AirFan4Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan4Rpm_CALC', kind='hinted')
+   AirFan5Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan5Rpm_CALC', kind='hinted')
+   AirFan6Rpm_Percent = Cpt(EpicsSignalRO, ':AirFan6Rpm_CALC', kind='hinted')
+   WaterTempIn_F = Cpt(EpicsSignalRO, ':WaterTempIn_CALC', kind='hinted')
+   WaterTempOut_F = Cpt(EpicsSignalRO, ':WaterTempOut_CALC', kind='hinted')
+   WaterFlowRate_gpm = Cpt(EpicsSignalRO, ':WaterFlowRate_CALC', kind='hinted')
+   AirTempInTop_F = Cpt(EpicsSignalRO, ':AirTempInTop_CALC', kind='hinted')
+   AirTempInMid_F = Cpt(EpicsSignalRO, ':AirTempInMid_CALC', kind='hinted')
+   AirTempInBot_F = Cpt(EpicsSignalRO, ':AirTempInBot_CALC', kind='hinted')
+   AirTempOutTop_F = Cpt(EpicsSignalRO, ':AirTempOutTop_CALC', kind='hinted')
+   AirTempOutMid_F = Cpt(EpicsSignalRO, ':AirTempOutMid_CALC', kind='hinted')
+   AirTempOutBot_F = Cpt(EpicsSignalRO, ':AirTempOutBot_CALC', kind='hinted')
+   FrontDoorAccess = Cpt(EpicsSignalRO, ':FrontDoorAccess_CALC', kind='hinted')
+   RearDoorAccess = Cpt(EpicsSignalRO, ':RearDoorAccess_CALC', kind='hinted')
+
```

Comment:
We can probably cut this file's repetition down a lot with some inheritance. A lot of these devices are just "that existing device, but more", which can be expressed as:
```
class Device1ButMore(Device1):
    more = Cpt(...)
```
Sometimes this makes the file harder to read and debug, so it's a judgement call as to when it's appropriate.